internal/server/server.go

```diff
@@ -579,12 +585,13 @@ func (c *controller) Start() {
 
 	router := gin.Default()
 
-	router.Use(CORSMiddleware())
+	backend := router.Group("/")
+	// backend.Use(CORSMiddleware())
```

Comment:
Has some reason to comment this?

Reply:
I believe when serving static files from API, the origin will be the same, and the API accessible by /api.